mlir/include/mlir/Dialect/Linalg/Transforms/Transforms.h

```diff
@@ -1692,6 +1706,11 @@ void populateTransposeMatmulPatterns(RewritePatternSet &patterns,
 void populateBlockPackMatmulPatterns(RewritePatternSet &patterns,
                                      const ControlBlockPackMatmulFn &controlFn);
 
+/// Patterns to apply Winograd Conv2D algorithm.
```

Comment:
What's the difference between `2DPatterns` and `2DRewritePattersn`?

Reply:
I rename `2DRewritePatterns` to `DecomposeWinogradOpsPatterns`.